### ISSUE TITLE
"0" scale

### ISSUE THREAD

**Issue by TuxedoLad** [TheCymaera/minecraft-spider#26]
I know this isn't really an issue, but someone can come across this by accident and not know what to do. When doing /scale, setting it to 0 crashes the plugin and reqires a server restart.

**Comment by CuddlyCornet645** [comment]
Yes, this is a real issue. I am having the same "Issue" but it kinda makes sence that you should NOT type 0 but maybe add a message or something.

**Comment by WiseDodge** [comment]
Preferably set a hard-coded limit and range of the scale command. Similar to how the /effect command's duration parameter has a requirement "Integer must not be less than 1, found at 0 at position . . ."

Perhaps not lower than 0.07 (from my testing, the scale going below 0.07 hinders the spider dysfunctional.)
